jerry-core/ecma/builtin-objects/typedarray/ecma-builtin-typedarray-prototype.c

```diff
@@ -940,7 +940,11 @@ ecma_builtin_typedarray_prototype_set (ecma_value_t this_arg, /**< this argument
     source_length_num = 0;
   }
 
-  uint32_t source_length_uint32 = ecma_number_to_uint32 (source_length_num);
```

Comment:
`ECMA_OP_TO_NUMBER_TRY_CATCH`(936) should be substituted with `ecma_op_to_length` .